src/ipips/ipip-0445.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "IPIP-0445: trustless gateway skip-leaves option"
+date: 2023-10-09
+ipip: open
+editors:
+  - name: Hugo VALTIER
+    github: Jorropo
+    url: https://jorropo.net/
+    affiliation:
+        name: Protocol Labs
+        url: https://protocol.ai/
+relatedIssues:
+  - https://github.com/ipfs/specs/issues/444
+order: 445
+tags: ['ipips']
+---
+
+## Summary
+
+Introduce `skip-leaves` flag for the :cite[trustless-gateway].
```

Comment:
_(warning: obligatory rename thread ;)_

A slight concern here is that **the description does not match what the name suggests** :upside_down_face: 
It is not skipping _all_ leaves, only `raw` ones. 

Either we really skip all leaves (and have spec define them clearly, e.g. nodes without children / only with link to parent) or rename this to reflect what is really happening (skipping  raw blocks).

:point_right: Would either of the below be acceptable?
- `skip-raw-blocks=y|n` 
- `skip-raw=y|n`

Reply:
Switched to `skip-raw-blocks`